test/pod.bats

```diff
@@ -390,3 +390,33 @@ function teardown() {
 	output=$(crictl exec --sync "$ctr_id" ls -ld /etc)
 	[[ "$output" == *"test test"* ]]
 }
+
+@test "verify RunAsGroup in container" {
+  start_crio
+
+  jq '
+    .linux.security_context.run_as_user = { value: 1000 }
+    | .linux.security_context.run_as_group = { value: 1001 }
```

Comment:
doing this in the sandbox won't really change anything (the pause container may get it, but it won't do anything with it). I thinhk you can just use sandbox_config.json here

Reply:
Using different RunAsGroup values at pod and container level to highlight that container specific one gets picked.